### ISSUE TITLE
Research adding/modifying input channels

### ISSUE THREAD

**Issue by aamster** [AllenInstitute/DeepCell#53]
Given the current architecture of classifying ROIs, exploring the input channels might be the easiest way to improve model performance. Currently, we're using denoised max projection, denoised correlation projection, suite2p mask.

Previously, we tried swapping denoised max activation for denoised correlation projection and saw the same overall performance. See: http://mlflo-mlflo-16mqjx084gpy-1597208273.us-west-2.elb.amazonaws.com/#/compare-runs?runs=[%228ba77f2741ca4690b079d172747b4253%22,%2248e9c54de51e49698c68cdd7565d21a5%22]&experiment=8

But we didn't try adding more than 3 channels or explaining the importance of the channels in a different way.

Other ideas for channels:
- denoised max activation
- noisy max activation

The max activations are interesting because they are raw, unmodified frames from the video, compared to the projections which are highly "engineered"

Since we're using a model pretrained on imagenet, to add more than 3 input channels will require adding untrained weights to the kernel in the first layer.

We should collect metrics on which set of input channels perform best. 

We can also use techniques from explainable AI, such as shapley values, gradient descent into input, etc to see the impact/importance of different channels.
